Diagnostics/Get-VSwitchPathCost.ps1

```diff
@@ -127,34 +136,25 @@ function Get-CounterAverage([String[]] $statArray)
 
 if (IsHyperThreadingEnabled)
 {
-    Log "ERROR: Hyper-Threading is enabled. Please disable Hyper-Threading before starting the test."
+    Log "WARNING: Hyper-Threading is enabled. This may affect the accuracy of the results."
 }
 
-$switch = Get-VMSwitch $SwitchName
```

Comment:
This cmdlet is the only part of the script that requires admin, so I removed it in favor of checking if the counters exist. However, I don't check for duplicate switch names now.